src/dotnet/APIView/APIViewWeb/DTOs/CommentUpdatesDto.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text.Json.Serialization;
+using APIViewWeb.LeanModels;
+
+namespace APIViewWeb.DTOs
+{
+    public enum CommentThreadUpdateAction
+    {
+        CommentCreated = 0,
+        CommentTextUpdate,
+        CommentResolved,
+        CommentUnResolved,
+        CommentUpVoteToggled,
+        CommentDeleted
+    }
+
+    public class CommentUpdatesDto
+    {
+        [JsonPropertyName("commentThreadUpdateAction")]
+        public CommentThreadUpdateAction CommentThreadUpdateAction { get; set; }
+        [JsonPropertyName("nodeId")]
+        public string NodeId { get; set; }
+        [JsonPropertyName("nodeIdHashed")]
+        public string NodeIdHashed { get; set; }
+        [JsonPropertyName("reviewId")]
+        public string ReviewId { get; set; }
+        [JsonPropertyName("revisionId")]
+        public string RevisionId { get; set; }
+        [JsonPropertyName("commentId")]
+        public string CommentId { get; set; }
```

Comment:
These properties are already part of `CommentItemModel`. Can we avoid having the duplicate and use the information from existing model object?

Reply:
The main comment will not be present in all cases.